integration-tests/capacity/replenishment.test.ts

```diff
@@ -126,7 +123,7 @@ describe("Capacity Replenishment Testing: ", function () {
       const [stakeKeys, stakeProviderId] = await createAndStakeProvider("TinyStake", 1n*1000n*1000n);
       // new user/msa stakes to provider
       const userKeys = createKeys("userKeys");
-      await fundKeypair(devAccounts[0].keys, userKeys, 2n*1000n*1000n);
+      await fundKeypair(devAccounts[0].keys, userKeys, 2n*1000n*1000n + 100n*1000n*1000n);
```

Comment:
a little easier to read
```suggestion
      await fundKeypair(devAccounts[0].keys, userKeys, 102n*1000n*1000n);
```